TemplateApplication/Onboarding/OnboardingFlow.swift

```diff
@@ -34,7 +34,6 @@ struct OnboardingFlow: View {
         return healthKit.isFullyAuthorized
     }
     
-    
     var body: some View {
         OnboardingStack(onboardingFlowComplete: $completedOnboardingFlow) {
```

Comment:
if we're already updating the TemplateApplication to the next major release of HealthKitOnFHIR, we might also want to look into updating it to the next major release of SpeziOnboarding (which also has been tagged as an alpha release for a while now, and has been working fine in MHC. i'll look into whether there's anything we need to change there before tagging a new major release)

Reply:
It would be great to tag a next release in SpeziOnboarding in the near future and also bing it here; I might see this as a separate step. We already have a stable 1.0.0 and we should make this a separate integration point once we have tagged a 2.0.0 there and ideally tagged a relatively stable version (e.g. 1.0.0) for SpeziConsent.

Also created an issue for this here: https://github.com/StanfordSpezi/SpeziOnboarding/issues/72